pkg/kf/commands/utils/utils.go

```diff
@@ -122,3 +125,39 @@ func parseCommandLine() (args []string, flags map[string][]string) {
 	}
 	return args, flags
 }
+
+// CreateProxy creates a proxy to the specified gateway with the specified host in the request header.
+func CreateProxy(w io.Writer, host, gateway string) *httputil.ReverseProxy {
```

Comment:
Why are we exporting this function?

Reply:
it's being used in both apps and routes now, I'll move this over to the internal utils package in a separate PR